src/Android/Xamarin.Android/Samples/Network Analysis/ClosestFacility/ClosestFacility.cs

```diff
@@ -0,0 +1,199 @@
+// Copyright 2018 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an 
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific 
+// language governing permissions and limitations under the License.
+
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Symbology;
+using Esri.ArcGISRuntime.Tasks.NetworkAnalysis;
+using Esri.ArcGISRuntime.UI;
+using Esri.ArcGISRuntime.UI.Controls;
+using System;
+using System.Collections.Generic;
+using Android.App;
+using Android.OS;
+using Android.Widget;
+
+namespace ArcGISRuntime.WPF.Samples.ClosestFacility
+{
+    [Activity(Label = "ClosestFacility")]
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        "Closest Facility (Interactive)",
```

Comment:
Esri's style is to apply sentence case, even to titles, so this should become 'Closest facility (interactive)'.